Misc/NEWS.d/next/Library/2025-03-09-11-01-00.gh-issue-83461.auwd13.rst

```diff
@@ -0,0 +1 @@
+:func:`datetime.datetime.strptime` now only accepts ASCII input.
```

Comment:
This is a new feature that needs a What's New + a versionchanged in the corresponding docs. This should also affect time.strptime I think